libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/FeatureConfigurationFlyout.tsx

```diff
@@ -0,0 +1,334 @@
+
+interface IFeatureConfigurationFlyoutProps {
+  isOpen: boolean;
+  onDismissFlyout: () => void;
+  selectedFeatures: number[];
+  numberOfContinuousFeatureBins: { [featureIndex: number]: number };
+  updateSelectedFeatures: (
+    newSelectedFeatures: number[],
+    numberOfContinuousFeatureBins: { [featureIndex: number]: number }
+  ) => void;
+}
+
+interface IFeatureConfigurationFlyoutState {
+  newlySelectedFeatures: number[];
+  newNumberOfContinuousFeatureBins: { [featureIndex: number]: number };
+  items: IFeatureConfigurationRow[];
+}
+
+interface IFeatureConfigurationRow {
+  key: string;
+  featureName: string;
+  featureRemark: string;
+  groups: string[];
+  continuousFeatureBinningEnabled: boolean;
+}
+
+const maxFeatureBins = 10;
+const minFeatureBins = 2;
+
+export class FeatureConfigurationFlyout extends React.Component<
+  IFeatureConfigurationFlyoutProps,
+  IFeatureConfigurationFlyoutState
+> {
+  public static contextType = ModelAssessmentContext;
+  public context: React.ContextType<typeof ModelAssessmentContext> =
+    defaultModelAssessmentContext;
+  private _selection: Selection;
+
+  constructor(props: IFeatureConfigurationFlyoutProps) {
+    super(props);
+
+    this._selection = new Selection({
+      onSelectionChanged: () => {
+        const selectedIndices = this._selection.getSelectedIndices().slice();
+        this.setState({ newlySelectedFeatures: selectedIndices });
+      }
+    });
+
+    this.state = {
+      items: [],
+      newlySelectedFeatures: this.props.selectedFeatures,
+      newNumberOfContinuousFeatureBins: this.props.numberOfContinuousFeatureBins
+    };
+    this.updateSelection();
+  }
+
+  componentDidMount() {
+    this.setState(
+      { items: this.getItems(this.state.newNumberOfContinuousFeatureBins) },
+      () => {
+        this.updateSelection();
+      }
+    );
+  }
+
+  componentDidUpdate(prevProps: IFeatureConfigurationFlyoutProps) {
+    if (
+      this.props.selectedFeatures.length !==
+        prevProps.selectedFeatures.length ||
+      this.props.selectedFeatures.some(
+        (featureIndex, index) =>
+          featureIndex !== prevProps.selectedFeatures[index]
+      )
+    ) {
+      this.setState({ newlySelectedFeatures: this.props.selectedFeatures });
+    }
+  }
+
+  public render(): React.ReactNode {
+    const columns: IColumn[] = [
+      {
+        key: "featureName",
+        maxWidth: 350,
+        minWidth: 200,
+        name: localization.ModelAssessment.ModelOverview.featureConfiguration
+          .featureColumnHeader,
+        onRender: this.renderFeatureColumn
+      },
+      {
+        key: "groups",
+        minWidth: 300,
+        name: localization.ModelAssessment.ModelOverview.featureConfiguration
+          .groupsColumnHeader,
+        onRender: this.renderGroups
+      }
+    ];
+
+    return (
+      <Panel
+        isOpen={this.props.isOpen}
+        closeButtonAriaLabel="Close"

```

Comment:
arial labels need localize as well.

Reply:
Oh! We're not doing that anywhere (hundreds of places in the codebase). How about I open an item for this because we'll have to do that everywhere in the code. Better to take care of it in one big change that doesn't include other things.

Reply:
Done